src/Arcus.Observability.Tests.Unit/Correlation/TestCorrelationInfoAccessor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Arcus.Observability.Correlation;
+
+namespace Arcus.Observability.Tests.Unit.Correlation 
+{
+    /// <summary>
+    /// Test <see cref="ICorrelationInfoAccessor{TCorrelationInfo}"/> implementation to access the <see cref="TestCorrelationInfo"/> model
+    /// </summary>
+    public class TestCorrelationInfoAccessor : ICorrelationInfoAccessor<TestCorrelationInfo>
+    {
+        /// <summary>
+        /// Gets or sets the current correlation information initialized in this context.
+        /// </summary>
+        public TestCorrelationInfo CorrelationInfo
+        {
+            get => throw new NotImplementedException();
```

Comment:
🤔

Reply:
Just for testing. In the right terminology it would be called a "dummy".http://xunitpatterns.com/Dummy%20Object.html

Reply:
Do you want me to change it to a regular property?

Reply:
Since you're not using it I would use a regular property instead indeed